src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -193,13 +193,13 @@ type SynMeasure =
     | Seq of measures: SynMeasure list * range: range
 
     /// A division of two units of measure, e.g. 'kg / m'
-    | Divide of measure1: SynMeasure * measure2: SynMeasure * range: range
+    | Divide of measure1: SynMeasure option * measure2: SynMeasure * range: range
 
     /// A power of a unit of measure, e.g. 'kg ^ 2'
     | Power of measure: SynMeasure * power: SynRationalConst * range: range
```

Comment:
I guess ranges inside the rational consts are also needed.

Reply:
Good morning @auduchinok. Are you okay with the ranges we now have inside of `SynRationalConst`?
I think, we don't need anything for `SynRationalConst.Negate` as `SynRationalConst.Rational` has the overall range and now also the partial ranges of `numerator`, `denominator`.

Reply:
Please capture the range of `Negate` case as well. Using `lhs` for capturing ranges doesn't work well in recovery rules, so if we add them, we'll need to construct the ranges from the ones captures in the children rules.